interface/modules/custom_modules/oe-module-faxsms/index.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018-2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+$sessionAllowWrite = true;
+require_once(__DIR__ . "/../../../globals.php");
+
+use OpenEMR\Modules\FaxSMS\Controller\AppDispatch;
+
+$serviceType = null;
+$action = $_REQUEST['_ACTION_COMMAND'] ?? null;
+$route = explode('/', $action);
+if (count($route ?? []) === 2) {
+    $serviceType = $route[0];
+    $action = $route[1] ?: $action;
+}
+$serviceType = $serviceType ?: $_REQUEST['type'];
+AppDispatch::getApiService($serviceType);
```

Comment:
Is this supposed to return a service class like I see in several other spots of your code, or does this echo html?  Threw me for a loop as I thought something was broken here as I didn't see anything done with the return.

Reply:
nope should be setApiService()(forgot the method also ohh if only you could be in my head!) it got jumbled when I was refactoring from using /fax/ or /sms/ as a rest api call instead of setting a service type as a query.
Because of the way we locked down calling the directory to redirect I abandoned, although, this will pose a problem for future modules. We got around this issue with oAuth by double down on security so we could expose ignoreAuth.
I really wanted this to be a full rest interface but I'm too lazy anymore to do it correctly!:)
nice catch.
Thanks for reviewing.